workflow-runtime/src/test/java/com/squareup/workflow1/RenderWorkflowInTest.kt

```diff
@@ -151,6 +153,35 @@ class RenderWorkflowInTest {
     assertEquals("updated state", restoredRenderings.value.rendering.first)
   }
 
+  // https://github.com/square/workflow-kotlin/issues/223
+  @Test fun `snapshots are lazy`() {
+    lateinit var sink: Sink<String>
+    var snapped = false
+
+    val workflow = Workflow.stateful<Unit, String, Nothing, String>(
+        initialState = { _, _ -> "unchanging state" },
+        snapshot = { Snapshot.of {
+          // We need to use snapped b/c the fail call is swallowed.
+          // https://github.com/square/workflow-kotlin/issues/223
```

Comment:
Nit: I think you linked the wrong issue here, did you mean #224?

Reply:
Yup, thanks.